app/templates/index.html

```diff
@@ -5,7 +5,7 @@
     <main class="govuk-main-wrapper">
         <h1 class="govuk-heading-l">Upcoming masterclasses</h1>
 
-        {% for masterclass in masterclasses %}
+        {% for masterclass in masterclasses if not masterclass.draft %}
```

Comment:
Why do you filter in the template and not on the server?

Reply:
So I couldn't find a way of doing it in SQLAlchemy so got all Masterclasses and then used a for loop. Have I missed something obvious?

```
def index():
    all_masterclasses = Masterclass.query.order_by(Masterclass.timestamp.asc()).all()
    masterclasses = []
    for m in all_masterclasses:
        if not m.draft:
            masterclasses.append(m)
    return render_template('index.html', title='Home', user=User, masterclasses=masterclasses)
```

Reply:
Could you use `filter_by(Masterclass.draft is None)` in your query?

Reply:
And, if not, there's a neat one-liner for your filter - you can compress four lines down to
`masterclasses = [masterclass for masterclass in all_masterclasses if masterclass.draft is not None]`

Reply:
The `filter_by(draft=None)` worked 👍